modules/howtos/examples/transactions-example.php

```diff
@@ -56,81 +55,88 @@ function removeOrWarn($collection, $doc) {
+} catch (\Couchbase\Exception\TransactionCommitAmbiguousException $e) {
+  echo "Transaction possibly committed: $e\n";
 }
 
+echo "Running: create example\n";
 // tag::create[]
 try {
   $cluster->transactions()->run(
     function (TransactionAttemptContext $ctx) {
-    // `$ctx` is a TransactionAttemptContext which permits getting, inserting,
-    // removing and replacing documents, performing N1QL queries, etc.
-    // … Your transaction logic here …
-    // Committing is implicit at the end of the lambda.
-    });
-}
-catch (\Couchbase\Exception\TransactionOperationFailedException $e) {
-    echo "Transaction did not reach commit point: $e\n";
-}
-catch (\Couchbase\Exception\TransactionException $e) { 
-// TODO check is this equivalent to TransactionCommitAmbiguousError as per Node examples?
+      // `$ctx` is a TransactionAttemptContext which permits getting, inserting,
+      // removing and replacing documents, performing N1QL queries, etc.
+      // … Your transaction logic here …
+      // Committing is implicit at the end of the lambda.
+    }
+  );
+} catch (\Couchbase\Exception\TransactionFailedException $e) {
+  echo "Transaction did not reach commit point: $e\n";
+} catch (\Couchbase\Exception\TransactionCommitAmbiguousException $e) {
   echo "Transaction possibly committed: $e\n";
 }
 // end::create[]
 
+echo "\nRunning: examples\n";
 // tag::examples[]
-
 try {
   $cluster->transactions()->run(
     function (TransactionAttemptContext $ctx) use ($collection) {
       // Inserting a doc:
-      $ctx->insert($collection, 'doc-a', []);
+      $ctx->insert($collection, 'doc-c', []);
 
       // Getting documents:
       $docA = $ctx->get($collection, 'doc-a');
 
       // Replacing a doc:
       $docB = $ctx->get($collection, 'doc-b');
-      $content = $docB->content;
+      $content = $docB->content();
       $newContent = array_merge(
         ["transactions" => "are awesome"],
-        $content);
-      
+        $content
+      );
       $ctx->replace($docB, $newContent);
 
       // Removing a doc:
       $docC = $ctx->get($collection, 'doc-c');
       $ctx->remove($docC);
 
-      // Performing a SELECT N1QL query against a scope:
-      $qr = $ctx->query('SELECT * FROM hotel WHERE country = $1',
-        [ "scope" => "inventory",
-          "parameters" => ["United Kingdom"]
-      ]);
-      echo $qr->rows();
-
-      $ctx->query('UPDATE route SET airlineid = $1 WHERE airline = $2',
-        [ "scope" => "inventory",
-          "parameters" => ['airline_137', 'AF'] ] );
-    });
-}
-catch (\Couchbase\Exception\TransactionOperationFailedException $e) {
-    echo "Transaction did not reach commit point: $e\n";
-}
-catch (\Couchbase\Exception\TransactionException $e) { 
-// TODO check is this equivalent to TransactionCommitAmbiguousError as per Node examples?
+      // Performing a SELECT N1QL query:
+      $selectQuery = 'SELECT * FROM `travel-sample`.inventory.hotel WHERE country = $1 LIMIT 5';
+      $qr = $ctx->query(
```

Comment:
> "it returns the same QueryResult you are used to"
> I hope this is actually a TransactionQueryResult...

@programmatix in response to your comment in the JIRA ticket, from what I can see `ctx->query(...)` returns a `QueryResult`:
https://github.com/couchbase/couchbase-php-client/blob/4.0.0/Couchbase/TransactionAttemptContext.php#L128-L140 

@avsej maybe you can expand on this, but seems the expectation is that this should be a `TransactionQueryResult`?

Node.js SDK seems to have this: https://docs.couchbase.com/sdk-api/couchbase-node-client/classes/TransactionAttemptContext.html#query